.env.example

```
@@ -42,4 +42,9 @@ BASE_URL=http://localhost:8080
 OPENAI_API_KEY=your_openai_api_key
 
 #10 AES
-AES_SECRET=your_aes_secret_key
\ No newline at end of file
+AES_SECRET=your_aes_secret_key
+
+# 11. Meta Marketing API
+META_APP_ID=your_meta_app_id
+META_APP_SECRET=your_meta_app_secret
+META_REDIRECT_URI=http://localhost:8080/api/meta/callback
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)